src/ovp_pipeline/commands/score_domain.py

```
@@ -0,0 +1,278 @@
+    parser.add_argument("--offline", action="store_true",
+                        help="Skip LLM call; emit heuristic stub for manual edit")
+    parser.add_argument("--force", action="store_true",
+                        help="Re-score even if host already in overrides")
+    args = parser.parse_args(argv)
+
+    vault = args.vault_dir.resolve()
+    if not vault.is_dir():
+        print(f"vault not found: {vault}", file=sys.stderr)
+        return 2
+
+    host = args.host.lower().lstrip("www.")
```

Ruff (0.15.12)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)